src/animators/animator.ts

```diff
@@ -1,5 +1,39 @@
 module Plottable {
+export module Animators {
+  export type d3SelectionOrTransition = d3.Selection<any>|d3.Transition<any>;
 
+  export type EasingFunction = (t: number) => number;
+  export type EasingFunctionSpecifier = string|EasingFunction;
+  export class EasingFunctions {
+    public static atStart = (t: number) => {
+      return 1;
+    };
+    public static atEnd = (t: number) => {
+      if (t < 1) {
+        return 0;
+      };
+      return 1;
+    };
+    public static squEase(easingFunction: EasingFunctionSpecifier, start: number, end: number): EasingFunction {
+      return (t: number) => {
+        if (start === undefined) {
+          start = 0;
+        };
+        if (t >= end) {
+            return 1;
+        }
+        if (t <= start) {
+          return 0;
+        }
+        let tbar = (t - start) / (end - start);
+        if (typeof (easingFunction) === "string") {
+          easingFunction = d3.ease(<string>easingFunction);
+        }
+        return (<EasingFunction>easingFunction)(tbar);
+      };
+    }
+  }
```

Comment:
Yes, these EasingFunctions could be used with the ```Easing``` animator. An easing function is just a function e(t) from [0,1] to [0,1] that specifies how much of the animation is applied at time t.
d3 has a whole lot of these known by name, but more generally, you should be able to use a predefined one by name, or pass a function. 
This is why I added the types ```EasingFunction``` and ```EasingFunctionSpecifier```.